pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -70,7 +71,7 @@ func createUpdateRuntimeObjects(cp *CloudPlatform) []runtime.Object {
 	}
 	updateRTObjects := []runtime.Object{csfs, csrbd, cfs, cbp}
 
-	if !isValidCloudPlatform(cp.platform) {
+	if !shouldSkipCephObjectDeployment(cp.platform) {
```

Comment:
This rename is not valid, since that is not what it is testing. Just do `isValidPlatform()`

Reply:
Done!